docs/algorithms/Evaluation.asciidoc

```diff
@@ -26,9 +26,11 @@ The steps involved are as follows:
 The final score is a value of 0 to 1 and is calculated as the following equation where sigma denotes the sum of all pixels in the raster:
 
 [[Equation1]]
-.Equation 1
-
-image::images/image003.png[scale="75"]
+.Completeness Score
+[latexmath]
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+\[ \frac{ \sum Absolute Difference }{ \sum Blurred 1 + \sum Blurred 2 } \]
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
There were a lot of terribly grainy images of equations that I removed and redid in latexmath.  Now they are able to be selected in the PDF and look better.